qingstor/qsctl/commands/base.py

```diff
@@ -254,8 +256,25 @@ def main(cls, args):
                 cls.client = cls.get_client(conf)
                 break
 
+        cls._init_recorder()
+
         if cls.client is None:
             sys.exit(-1)
 
         # Send request
         return cls.send_request(options)
+
+    @classmethod
+    def _init_recorder(cls):
+        # Init UploadIdRecorder
+        qsctl_dir = os.path.expanduser("~/.qingstor/qsctl")
```

Comment:
Don't hardcode ~/.qingstor/qsctl, user may specify other path for config in command line, maybe we need create qsctl_dir under config file.